source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -40,9 +40,11 @@
 // Constructor for class GLSNavierStokesSolver
 template <int dim>
 GLSSharpNavierStokesSolver<dim>::GLSSharpNavierStokesSolver(
-  SimulationParameters<dim> &p_nsparam)
-  : GLSNavierStokesSolver<dim>(p_nsparam)
+  CFDDEMSimulationParameters<dim> &p_nsparam)
+  : GLSNavierStokesSolver<dim>(p_nsparam.cfd_parameters)
+  , cfd_dem_parameters(p_nsparam)
```

Comment:
it's a bit redundant. Why the two attributes?

Reply:
The parameter of the GLSNavierStokesSolver only takes the CFD part of the parameter and the cfd_dem_parameters need the full set of the parameter to be initialized.